NAS-101963 doc that replication task names are unique (#1296)
freenas/freenas-docs: userguide/tasks.rst

This first screen provides options to configure the connection between
|rpln-sys1| and |rpln-sys2|.

> Enter a descriptive :guilabel:`Name` for this replication. Replication

Reviewer: As above: s/Replication/Replication Task/
```suggestion
Enter a descriptive :guilabel:`Name` for this replication task. Replication Task
```

> names, SSH Connection names, and SSH Keypair names must be unique.

Reviewer: Why are we mentioning here that SSH connections and keypair names also have to be unique?  Does that relate to this section, or should it be mentioned elsewhere when those names are discussed?

Author: Good point. I forgot to update those specific sections. I'll update them now.

However, I still think it's relavent here as well because SSH connections or SSH key pairs may be needed to set up a replication task.

This first screen provides options to configure the connection between
|rpln-sys1| and |rpln-sys2|.

Enter a descriptive :guilabel:`Name` for this replication. Replication

> names are unique. They also cannot be shared by connection and

Reviewer: The "connection" and "keypair" words should be :ref:s to the those sections.

```suggestion
names must be unique. They also cannot be shared by connection or
```

|rpln-sys1| and |rpln-sys2|.

Enter a descriptive :guilabel:`Name` for this replication. Replication

> names, SSH Connection names, and SSH Keypair names are unique.

Reviewer: As above.
